CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # Release History
-## 1.0.0 (2012-11-01)
+
+## 1.1.0 (2023-01-19)
+
+### Behavior Changes
+- `SnowparkSourceCodeExtractionException` would be raised when Snowpark fails to extract the source code of a function referenced by UDF registration.
+- `Session.create_dataframe()` no longer ignores the optional parameter `schema` when `data` is a pandas DataFrame.
```

Comment:
should we be more specific about how schema works here? For example, we can say `Session.create_dataframe()` will use `schema` to build the schema of the dataframe if it's provided now when `data` is a pandas DataFrame. (my words might not be accurate)